footylib/footylib.py

```diff
@@ -42,13 +43,15 @@ def __front_page(self):
     def competitions(self):
         if not self._competitions:
             locations = self.__front_page.find_all('div',
-                                                 {'class': 'fusion-panel panel-default'})
+                                                 {'class': 'fusion-panel '
+                                                           'panel-default'})
             for location_data in locations:
                 location = location_data.find('div',
                                               {'class': 'fusion-toggle-heading'}
                                               ).text
                 for competition in location_data.find_all('a',
-                                                          {'class': 'footycombut'}):
+                                                          {'class': 'footy'
```

Comment:
maybe breaking this down is not the best thing to do since it is one single word actually. That would be confusing when you are  trying to match it with the html code.